gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -478,7 +507,8 @@ MM_ParallelGlobalGC::mainThreadGarbageCollect(MM_EnvironmentBase *env, MM_Alloca
 			_collectionStatistics._tenureFragmentation |= MACRO_FRAGMENTATION;
 		}
 
-		mainThreadCompact(env, allocDescription, rebuildMarkBits);
+		// mainThreadCompact(env, allocDescription, rebuildMarkBits);
+		mainThreadCompact(env, allocDescription, true);
```

Comment:
this should be conditional for explictGC only (and for the final solution for checkpoint GC only)

Reply:
actually should something like 'rebuildMarkBits || explicit GC'